client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -491,6 +491,15 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		}
 	}
 
+	.generated-design-picker__previews-scaled {
+		position: absolute;
```

Comment:
This one makes the height of the parent of the previews is incorrect and it leads to the wrong footer position 😅